### ISSUE TITLE
CI blocked on `test_broken_worker_during_computation`

### ISSUE THREAD

**Issue by jacobtomlinson** [dask/distributed#4415]
Looks like CI is still blocked with `distributed/tests/test_failed_workers.py::test_broken_worker_during_computation` failing on Travis only.

**Comment by fjetter** [comment]
This is a duplicate of https://github.com/dask/distributed/issues/4173

**Comment by jacobtomlinson** [comment]
Good spot thanks @fjetter
